man/man7/zfsprops.7

```diff
@@ -1017,6 +1017,41 @@ See the
 section of
 .Xr zfsconcepts 7 .
 .It Xo
+.Sy direct Ns = Ns Sy disabled Ns | Ns Sy standard Ns | Ns Sy always
+.Xc
+Controls the behavior of direct requests (e.g. O_DIRECT). The
+.Sy standard
+behavior for direct requests is to bypass the ARC when possible.
+These requests will not be cached and performance will be limited by the
+raw speed of the underlying disks (this is the default).
+.Sy always
+causes every properly aligned read or write to be treated as a direct request.
+.Sy disabled
+causes the O_DIRECT flag to be silently ignored and all direct requests will
+be handled by the ARC.
+This is the default behavior for the OpenZFS 2.0 release and prior.
```

Comment:
"for the OpenZFS 2.0 and prior releases"

Also, O_DIRECT probably wants to be `.Dv` here and below?

Reply:
Updated.